recipes/ddtrace/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ddtrace" %}
+{% set version = "0.42.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ddtrace-{{ version }}.tar.gz
+  sha256: dba66c7013a4352dfa20ed31e989001250056c89811d1cb5ee10e23dc98bac0a
+
+build:
+  number: 0
+  entry_points:
+    - ddtrace-run = ddtrace.commands.ddtrace_run:main
+    - pyddprofile = ddtrace.profiling.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+    - setuptools_scm
+    - setuptools_scm >=4
+  run:
+    - enum34
+    - funcsigs >=1.0.0
+    - intervaltree
+    - protobuf >=3
+    - python
+    - tenacity >=5
+
+test:
+  imports:
+    - ddtrace
+    - ddtrace.bootstrap
+  commands:
+    - pip check
+    - ddtrace-run --help
```

Comment:
doesn't look like you can just run these tests here.  I'd remove them for now

Reply:
Thanks Marius, looks like you're right :)